specs/latest/2.0/index.html

```diff
@@ -3443,6 +3443,21 @@ <h3>Preventing undefined behavior with Transform Feedback</h3>
         OpenGL ES 3.0.5 &sect;2.15.2</a>)</span>
     </div>
 
+    <p>
+        In case the same buffer is bound to more than one transform feedback binding point that will
+        be written to, <code>BeginTransformFeedback</code> generates an
```

Comment:
I did some more experiments, it seems like ANGLE's DX backend fails to write the outputs correctly even when using two completely disjoint regions of the same output buffer for different varyings. On NVIDIA's GL driver it works. Don't know if it is something fundamental in DX or if it's an ANGLE bug. But it would be easy enough to amend the spec so that binding disjoint regions with bindBufferRange are allowed.